bve/src/parse/panel2_cfg/sections/this.rs

```diff
@@ -0,0 +1,35 @@
+use crate::HexColor3;
+use bve_derive::FromKVPSection;
+use cgmath::{Array, Vector2};
+
+#[derive(Debug, Clone, PartialEq, FromKVPSection)]
+pub struct ThisSection {
```

Comment:
I don't like this name but I guess it comes directly from BVE itself :<

Reply:
Yup, it's a section called `this`